pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -1094,6 +1094,11 @@ type NodeGroupBase struct {
 	// Defaults to `false`
 	// +optional
 	DisableIMDSv1 *bool `json:"disableIMDSv1,omitempty"`
+
+	// DisableIMDS disallows IMDS requests
```

Comment:
WDYT about "blocks IMDS requests"?

```suggestion
	// DisableIMDS blocks IMDS requests
```